packages/react-examples/.storybook/preview.js

```diff
@@ -5,6 +5,12 @@ import { withInfo } from '@storybook/addon-info';
 import { withPerformance } from 'storybook-addon-performance';
 import { withFluentProvider, withKeytipLayer, withStrictMode } from '@fluentui/storybook';
 
+/**
+ * "PACKAGE_NAME" placeholder is being replaced by webpack loader - @link {./preview.loader}
+ * @type {string}
+ */
+const packageName = 'PACKAGE_NAME';
```

Comment:
nit: now that you're moving it into a variable
```suggestion
const packageNamePlaceholder = 'PACKAGE_NAME';
```